packages/rsocket-core/src/ClientServerMultiplexerDemultiplexer.ts

```diff
@@ -138,7 +138,7 @@ export class ClientServerInputMultiplexerDemultiplexer
     this.registry[streamId] = stream;
 
     if (
-      !stream.handleReady(streamId, {
+      !stream.handleReady?.(streamId, {
```

Comment:
@OlegDokuka should `RequestResponseResponderStream` implement `handleReady`, or do we need to check for its existence here? Or does the execution reaching this point indicate some of problem potentially?

Reply:
This is not possible to be nulk since responder should never be involved in that execution path, only requesters. Thus no need ti perform these extra check since we will get a hanging execution in any case if this method is null

Reply:
#169 shows an exception is thrown from this site, so we may have something else going on then if a responder call path should never have reached here.

Reply:
Likely something is wrong there. I will have a look